packages/client/src/forms/correction/reason.ts

```diff
@@ -71,13 +73,16 @@ export const correctRecordReasonSectionGroup: IFormSectionGroup = {
             name: 'reasonForChange',
             type: 'TEXT',
             label: {
-              defaultMessage: 'Reason for change',
+              defaultMessage: 'Reason for correction',
```

Comment:
This translation needs to be changed in your farajaland PR as well

Reply:
updated